app/Http/Resources/CategoryResource.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class CategoryResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @return array<string, mixed>
+     */
+    public function toArray(Request $request): array
+    {
+        return parent::toArray($request);
```

Comment:
@Geri-PG You need to return the array of category attributes. Check laravel doc: [https://laravel.com/docs/11.x/eloquent-resources#concept-overview](url)